src/pages/docs/step-ca/webhooks.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Webhooks
+html_title: Webhooks
+description: Webhooks augment the data available to certificate templates
+---
+
+Administrators can attach webhooks to provisioners to retrieve additional data that will be available when rendering [certificate templates](/docs/step-ca/templates).
```

Comment:
On the other hand if webhooks in open source are really limited in terms of their use—if they really are just an extension of templates—then we should consider placing these docs under the Templates doc, rather than in a separate page.

Reply:
I see now that webhooks are useful for two purposes:
- enriching certs
- external policy evaluation

and maybe a third: metrics & auditing hooks

Reply:
Callout added in 53acabd05a61a131bba8be9a002b7188b7cb4a51. There was a product decision made to not document the policy/authorization capabilities of webhooks yet because it could create confusion of how that interacts with other features for enforcing policies. The default webhook kind is ENRICHING, which happens early and so includes the CSR. The other kind is AUTHORIZING and that is called at the end with the certificate to be signed.

Reply:
I see.